lib/dor/services/client/mutate.rb

```diff
@@ -27,15 +27,17 @@ def apply_admin_policy_defaults
 
         # Updates the object
         # @param [Cocina::Models::DRO,Cocina::Models::Collection,Cocina::Models::AdminPolicy] params model object
+        # @param [String] etag if provided, send this via an If-Match header. The server will use this to know if we've started with the latest version.
         # @raise [NotFoundResponse] when the response is a 404 (object not found)
         # @raise [UnexpectedResponse] when the response is not successful.
         # @return [Cocina::Models::DRO,Cocina::Models::Collection,Cocina::Models::AdminPolicy] the returned model
-        def update(params:)
+        def update(params:, etag: nil)
           resp = connection.patch do |req|
             req.url object_path
             req.headers['Content-Type'] = 'application/json'
             # asking the service to return JSON (else it'll be plain text)
             req.headers['Accept'] = 'application/json'
+            req.headers['If-Match'] = etag if etag
```

Comment:
I'm curious if you can say a little about expecting/requiring collaborators to inject the etag vs., say, computing the etag given the model in the `params` kwarg?

Reply:
i was also wondering about that

Reply:
The etag you provide is from the previous version. If we computed the etag from the updated version, then it would never match.